controllers/githubHooks.js

```diff
@@ -76,8 +76,11 @@ var HooksController = {
 
          // Update DB with new pull request content.
          dbUpdated = preUpdate.then(function() {
-            return dbManager.updatePull(new Pull(body.pull_request));
-         });
+            var pull = new Pull(body.pull_request);
+            // Update the pull with information from the related issue
+            return pull.syncToIssue();
```

Comment:
I think this logic may be reversed. The name is a bit hard to parse, and perhaps that's why. This function loads the *issue* from the DB and updates the *pull request* with issue data from the DB. To me, it seems like we need to do that when github tells us the issue is updated, not when github tells us the pull is.